src/cosmicqc/analyze.py

```diff
@@ -95,9 +168,85 @@ def find_outliers(
     return outliers_df[columns_to_include]
 
 
+def label_outliers(
+    df: pd.DataFrame,
+    feature_thresholds: Optional[Union[Dict[str, float], str]] = None,
+    feature_thresholds_file: Optional[str] = DEFAULT_QC_THRESHOLD_FILE,
+    include_threshold_scores: bool = True,
```

Comment:
The goal of this function seems to be to label the data frame with outliers and passing single-cells, so I am a bit confused about the function's purpose if the labeling parameter wasn't used.

Reply:
Thanks for this comment, I need to do a better job of documenting what's happening here.

What did you mean by the `labeling` parameter? I think I might be misunderstanding what you mean about that particular aspect.

`label_outliers` takes a DataFrame:

| Col_A | Col_B |
|----------|----------|
| A  | 1  |
| B   | 2  |

And creates a "labeled outlier" DataFrame:

| Column 1 | Col_B | Zscore Col_B | Outlier |
|----------|----------|----------|----------|
| A   | 1  | -1  | False  |
| B   | 2   | 1   | True   |

(assuming a threshold of 0.5)

Reply:
Thanks for providing more clarification! I think what I mean is `include_threshold_scores` is what labels the data frame with the outlier column so if someone can change that to False instead of True, what would the function do?

Reply:
Yup! Makes total sense now! I think that adding the z-scores might not totally be necessary, but I don't have a good enough argument one way or another. I default to not including so maybe the default is False and can be optionally turned on?

Reply:
Thanks! Sounds good, I'll change the default here to exclude the zscores as you mention.